cfgov/unprocessed/apps/find-a-housing-counselor/css/hud_print.css

```diff
@@ -54,136 +65,51 @@
         padding: 0;
     }
 
-    p,
-    h1,
-    h2,
-    h3,
-    h4 {
-        orphans: 3;
-        widows: 3;
-    }
-
-    h1,
-    h2,
-    h3,
-    h4 {
-        page-break-after: avoid;
-    }
-
-    /* Hide unneeded content */
-    .o-header,
-    .bread,
-    .share,
-    .o-footer,
-    .hud_hca_api_print_hide {
-        display: none !important;
-    }
-
-    /* Some adjustments */
-    table, thead, tbody, tfoot, tr, td, th, caption {
-        border: 0 !important;
-        display: block !important;
-        margin: 0 !important;
-        padding: 0 !important;
-    }
-
-    .pagetitle > h1 {
-        font-size: 16pt;
-    }
-
-    img.hud_hca_api_print_logo {
-        float: right;
-        max-width: 20% !important;
-    }
-
-    .cfpb_hud_hca_api_print_text {
-        border-bottom: 4px solid #101820;
-        padding-bottom: 1em;
-    }
-
-    .hud_hca_api_the_results {
-        font-weight: bold;
-        text-transform: uppercase;
-    }
-
-    /* Grid adjustments */
-    tr.hud_hca_api_result:before,
-    tr.hud_hca_api_result:after {
-        content: " ";
-        display: table;
-    }
-
-    tr.hud_hca_api_result:after {
-        clear: both;
-    }
-
-    tr.hud_hca_api_result {
-        *zoom: 1;
+    /* Prevent blank sheet at the end of the page. */
+    main,
+    .content_main {
+      margin-bottom: 0;
+      padding-bottom: 0;
     }
 
-    tr.hud_hca_api_result {
-        border-bottom: 1px solid #babbbd !important;
-        padding: 1.5em 0 !important;
+    /* Item result row. */
+    tr {
+      display: grid !important;
+      grid-template-columns: 2;
+      grid-template-rows: 2;
     }
 
-    .hud_hca_api_counselors_td,
-    .hud_hca_api_services_td {
-        float: left;
-        width: 88%;
+    /* Item services. */
+    td:nth-child(2) {
+      grid-column-start: 1;
+      grid-row-start: 2;
     }
 
-    .hud_hca_api_distances_td {
-        float: right;
-        margin-left: 2% !important;
-        margin-top: -255px !important;
-        text-align: right;
-        width: 10%;
+    /* Item distance. */
+    td:nth-child(3) {
+      grid-column-start: 2;
+      grid-row-start: 1;
     }
 
-    .hud_hca_api_counselor_address,
-    .hud_hca_api_site,
-    .hud_hca_api_tel,
-    .hud_hca_api_lang,
-    .hud_hca_api_serv,
-    .hud_hca_api_distance {
-        page-break-inside: avoid;
```

Comment:
Looks like the old version avoided page breaks inside results. Is that something we want to keep? Personally I kind of like avoiding page breaks there to keep all info about a result on the same page, but I don't feel super strongly either way.

Reply:
With this included it made one result per page. I think we wanted to stack them as close as possible?

Reply:
Yeah, I was worried something like one result per page might happen. I definitely prefer *not* one result per page over keeping the info for a listing together, so let's leave this one as is.